Dockerfile

```diff
@@ -20,28 +88,36 @@ ENV GPG_PWD_B2C7B0F5 changeme
 ENV GPG_PWD_D4CE5C2B changeme
 ENV GPG_PWD_EEC960A4 changeme
 
-ADD otrs-5.0.18.tar.gz /opt/
-RUN ln -s /opt/otrs-5.0.18 /opt/otrs
 RUN useradd -r -d /opt/otrs/ -c 'OTRS user' otrs && usermod -G nogroup otrs
 
 COPY Config.pm /opt/otrs/Kernel/Config.pm
 RUN /opt/otrs/bin/otrs.SetPermissions.pl /opt/otrs --otrs-user=otrs --web-group=www-data
 RUN /opt/otrs/bin/otrs.CheckModules.pl
 
-RUN a2dissite 000-default
-RUN ln -s /opt/otrs/scripts/apache2-httpd.include.conf /etc/apache2/conf.d/otrs.conf
-RUN echo "RedirectMatch ^/$ /otrs-web/" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv MYSQL_PORT_3306_TCP_ADDR" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv MYSQL_PORT_3306_TCP_PORT" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv MYSQL_USERNAME" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv MYSQL_PASSWORD" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv MYSQL_DATABASE" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv SMTP_PORT_25_TCP_ADDR" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv SMTP_PORT_25_TCP_PORT" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv GPG_PWD_50D7E35A" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv GPG_PWD_B2C7B0F5" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv GPG_PWD_D4CE5C2B" >> /etc/apache2/conf.d/otrs.conf
-RUN echo "PerlPassEnv GPG_PWD_EEC960A4" >> /etc/apache2/conf.d/otrs.conf
+RUN echo " \
+Listen *:80\n \
+LoadModule access_compat_module modules/mod_access_compat.so\n \
+LoadModule alias_module modules/mod_alias.so\n \
+LoadModule deflate_module modules/mod_deflate.so\n \
+LoadModule filter_module modules/mod_filter.so\n \
+LoadModule headers_module modules/mod_headers.so\n \
+LoadModule mime_module modules/mod_mime.so\n \
+LoadModule perl_module modules/mod_perl.so\n \
+LoadModule version_module modules/mod_version.so\n \
+RedirectMatch ^/$ /otrs-web/\n \
+PerlPassEnv MYSQL_PORT_3306_TCP_ADDR\n \
+PerlPassEnv MYSQL_PORT_3306_TCP_PORT\n \
+PerlPassEnv MYSQL_USERNAME\n \
+PerlPassEnv MYSQL_PASSWORD\n \
+PerlPassEnv MYSQL_DATABASE\n \
+PerlPassEnv SMTP_PORT_25_TCP_ADDR\n \
+PerlPassEnv SMTP_PORT_25_TCP_PORT\n \
+PerlPassEnv GPG_PWD_50D7E35A\n \
+PerlPassEnv GPG_PWD_B2C7B0F5\n \
+PerlPassEnv GPG_PWD_D4CE5C2B\n \
+PerlPassEnv GPG_PWD_EEC960A4\n \
+" >> /opt/otrs/scripts/apache2-httpd.include.conf
```

Comment:
can the file apache2-httpd.include.conf added using COPY? this would be easier to understand

Reply:
Done